config.c

```diff
@@ -339,7 +339,7 @@ static int readModeUidGid(const char *configFile, int lineNum, const char *key,
     }
 
     if (rc == 4) {
-        message(MESS_ERROR, "%s:%d extra arguments for "
+        message(MESS_WARN, "%s:%d extra arguments for "
```

Comment:
This seems wrong to me: we are not ignoring the extra arguments but abort parsing them all altogether. And all callers check the return value and call `RAISE_ERROR()` if it is `-1`.

Reply:
@cgzones You are right.  Thank you for spotting it!

@sblondon Could you please revert this change?

Reply:
Thanks for the update!